src/formatDate.js

```diff
@@ -9,6 +9,41 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+
+  const splitedDate = date.split(oldSeparator);
+
+  const newDate = [];
+
+  for (let i = 0; i < splitedDate.length; i++) {
+    const oldPart = fromFormat[i];
+
+    for (let j = 0; j < splitedDate.length; j++) {
+      const newPart = toFormat[j];
+
+      if (oldPart[0] === newPart[0]) {
+        let newDateElement = splitedDate[i];
+
+        newDateElement = splitedDate[i];
```

Comment:
This line is redundant as it duplicates the assignment from the previous line. You can remove it to clean up the code.